votingworks/arlo
# Record batch results

Task: #672 
- Add a new model BatchResult to record the audit results for a
batch+contest choice 
- Db migration to add BatchResult
- Add an endpoint for recording batch results for a jurisdiction for a
given round
- When results are recorded, check if the round is over, and if so,
calculate the risk measurements
- Add the recorded batch results to the batch list endpoint under `results` for each batch
